### DISCUSSION TITLE
SEB IOS crashing on loading task

### DISCUSSION THREAD

**Discussion by VarunTwri** [SafeExamBrowser/seb-mac#362] [General]
[
[SEB-2024-01-12-111101.txt](https://github.com/SafeExamBrowser/seb-mac/files/13915580/SEB-2024-01-12-111101.txt)
](url)

Please find the attached log for crash.

**Comment by danschlet** [comment]
Please also provide the SEB logs and full description (how to reproduce, expected, observed, see GitHub Issue template). Otherwise we cannot process this.

**Reply by VarunTwri** [reply]
Hi @danschlet  , attached seb logs for cra
[Uploading SEB-2024-01-12-111101.txt…]()
sh

**Reply by danschlet** [reply]
This log is not enough to analyze crashes AND ALWAYS A FULL WHAT WERE YOU DOING, WHAT DID YOU EXPECT, HOW TO REPRODUCE IS NECESSARY! If it's reproducible with specific SEB settings, provide these (including SEB settings, Quit and Admin passwords) or a link to a demo exam.

**Comment by danschlet** [comment]
Just create an issue and answer all the questions which the template displays. SEB logs can be exported by going to Settings app / SEB, selecting the according option and going back to SEB.
